### ISSUE TITLE
Backspace doesn't work when bold is activated

### ISSUE THREAD

**Issue by Edugolr** [froala/wysiwyg-editor#4378]
##### Expected behavior.
Backspace should delete characters before the cursor's current position.

##### Actual behavior.
Deleting with backspace while having bold activated doesn't delete characters before the cursor's current position.

##### Steps to reproduce the problem.

1. Go to https://froala.com/wysiwyg-editor/
2. Click inside the editor
3. Activate bold
4. Try to delete with backspace

Alternatively you can delete a bold text with backspace, then try to delete more by continuing to press backspace.

##### Editor version.
4.0.7

##### OS.
Ubuntu 20.04.3 LTS

##### Browser.
Version 96.0.4664.93 (Official Build) (64-bit)

##### Recording.
![froala-bold-bug](https://user-images.githubusercontent.com/30393363/146168434-0ebe3a9c-1cdb-44c5-985a-aa90d7ac8926.gif)

**Comment by Krisell** [comment]
We're facing this problem as well, and any formatting (italic, underline, color etc) has the same problem.

**Comment by ViOLeT-GiT** [comment]
Same problem.

**Comment by vilbergs** [comment]
Tested in 4.0.8, the issue still remains.

**Comment by macarenapoo** [comment]
Yeah, we're experiencing the same issue. If you type something in bold, then bold + italic, then just italic and try to delete, backspace is not going to work at some point 🤷‍♀️

**Comment by zahidkarim24** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
